test/requests.js

```diff
@@ -1066,7 +1066,7 @@ var tests = function(web3) {
     it("should return more than 0 accounts", function(done) {
       web3.eth.personal.getAccounts(function(err, result) {
         if (err) return done(err);
-        assert.equal(result.length, 3);
+        assert.equal(result.length, 13);
```

Comment:
Bit of a yak shave, this one...

In order to lock `accounts[1]` in the `transaction_rejection` tests, I had to fix the bug which prevents locking of the initially generated accounts. That is, that they aren't registered as personal accounts from the start. This in turn forced this change.